composer.json

```diff
@@ -21,15 +21,15 @@
         "ext-simplexml": "*",
         "aeon-php/calendar": "^1.0",
         "coduo/php-to-string": "^3",
-        "doctrine/lexer": "^1.0||^2.0"
+        "doctrine/lexer": "^1.0||^2.0||^3.0"
```

Comment:
as already mentioned, I would simplify it by allowing only "^3.0"

Reply:
yeah, but as I'm on doctrine orm 3.0 I'm forced to use lexer 3 😅 I'll keep the fork because it's "workish" and I can use it in my project, but i'll remove merge request in your repository, good luck with the fix ! (if ever I have the time I'll look into it)

Reply:
hmm but if you remove 1 and 2, it should work for you and I could merge it, no?

Reply:
ah sorry I haven't read your message properly, lexer 3.0 has a lot of BC Break and the code don't work with it :(

Reply:
yeah, I once tried to upgrade to 3.0 but it wasn't working and I didn't had then enough time to debug why but it seems that it wasn't a bug on my end but rather some changes in the internal behavior of lexer.
I'm gonna close this PR since it can't be merged anyway, sorry